pallets/author-inherent/src/lib.rs

```diff
@@ -208,10 +208,7 @@ impl<T: Config> ProvideInherent for Module<T> {
 			T::AccountId::decode(&mut &author_raw[..]).expect("Decodes author raw inherent data");
 		ensure!(
 			T::CanAuthor::can_author(&author),
-			InherentError::Other(sp_runtime::RuntimeString::Borrowed(format!(
```

Comment:
The &str in `RuntimeString::Bored(_)` had to have a static lifetime and `format` outputs a String with a temporary lifetime. I think the AccountId has a temporary lifetime so I just made it a &str directly.

Reply:
could probably clone if we want to print "{} cannot author" , I'm messing around with it now and found this https://stackoverflow.com/questions/23975391/how-to-convert-a-string-into-a-static-str